src/chrome/content/rules/Democrats.org.xml

```diff
@@ -0,0 +1,52 @@
+<!--
+	See also DNC.org.xml.
+
+	No working URL known:
+		- centrodevotacion.democrats.org
+		- convention.democrats.org
+		- ppapi.democrats.org
+
+	Time out:
+		- asdc.democrats.org
+		- developers.democrats.org
+		- directory.democrats.org
+		- factivists.democrats.org
+
```

Comment:
https://developers.democrats.org is already in the file so I'm not sure what you want me to do with that subdomain.